### PR TITLE
v.1.4. Shake to reload

### PR DESCRIPTION
**Pull request by vlast01**

### CODE REVIEW

File: RSSReader/View/FeedViewController/FeedViewController.m [hunk 1]
```diff
@@ -162,6 +165,32 @@ - (void)refreshTableViewCell:(int)index {
     } completion:nil];
 }
 
+- (void)motionEnded:(UIEventSubtype)motion withEvent:(UIEvent *)event {
+    if (motion == UIEventSubtypeMotionShake) {
```
Comment (AndreiChernyshou) [inline RSSReader/View/FeedViewController/FeedViewController.m]: It is preferable to use switch statement while working with enums

File: RSSReader/View/FeedViewController/FeedViewController.m [hunk 2]
```diff
@@ -162,6 +165,32 @@ - (void)refreshTableViewCell:(int)index {
...
+        [self showRefreshAlert];
+    }
+}
+
+- (void)showRefreshAlert {
+    UIAlertController *alert = [UIAlertController alertControllerWithTitle:NSLocalizedString(kReloadAlertMessage, nil)
```
Comment (AndreiChernyshou) [inline RSSReader/View/FeedViewController/FeedViewController.m]: This functionality should be in the extension

File: RSSReader/View/FeedViewController/FeedViewController.m [hunk 3]
```diff
@@ -25,6 +25,9 @@ @interface FeedViewController () <UITableViewDataSource, UITableViewDelegate, Cu
 NSString * const kErrorTitle = @"Error";
 NSString * const kErrorRetry = @"Retry";
 NSString * const kCellID = @"cellId";
+NSString * const kReloadAlertMessage = @"Reload feed?";
+NSString * const kYesActionTitle = @"Yes";
```
Comment (AndreiChernyshou) [inline RSSReader/View/FeedViewController/FeedViewController.m:29]: I would better use localised string keys for this.

**Review by AndreiChernyshou — CHANGES_REQUESTED** [review]

File: RSSReader/View/FeedViewController/FeedViewController.m (hunk 3 above)
Comment (vlast01) [reply RSSReader/View/FeedViewController/FeedViewController.m:29]: This constants are keys in NSLocalizedStrings

File: RSSReader/View/FeedViewController/FeedViewController.m (hunk 2 above)
Comment (vlast01) [reply RSSReader/View/FeedViewController/FeedViewController.m]: added

File: RSSReader/View/FeedViewController/FeedViewController.m (hunk 1 above)
Comment (vlast01) [reply RSSReader/View/FeedViewController/FeedViewController.m]: done

File: RSSReader/View/FeedViewController/FeedViewController.m (hunk 3 above)
Comment (AndreiChernyshou) [reply RSSReader/View/FeedViewController/FeedViewController.m:29]: Good.

**Review by AndreiChernyshou — APPROVED** [review]

**Pull request merged**